src/toil/job.py

```diff
@@ -982,9 +982,9 @@ def deleteLocalFile(self, fileStoreID):
                             os.remove(cachedFile)
                             cacheInfo.cached -= fileSize
                     self.logToMaster('Successfully deleted cached copy of file with ID '
-                                     '\'%s\'.' % fileStoreID)
+                                     '\'%s\'.' % fileStoreID, logging.DEBUG)
                 self.logToMaster('Successfully deleted local copies of file with ID '
-                                 '\'%s\'.' % fileStoreID)
+                                 '\'%s\'.' % fileStoreID, logging.DEBUG)
 
```

Comment:
+1 on changing these to standard logging.

@chapmanb, why did you change this? Depending on your response, we might ask you take undo these two changes out and we might address them in a second PR.